src/components/nav.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "gatsby";
+import * as navStyles from './nav.module.scss';
```

Comment:
tutaj zamiast navStyles dalbym po prostu styles, ale to też może być moja konwencja przypisywania klas z css modules. Więc nie sugeruj się tym na razie ;)